app/assets/stylesheets/app/slop.scss

```diff
@@ -134,47 +32,26 @@ $user-davidkaneda: #848C24;
   margin-top: $line-height-computed * 2;
   margin-bottom: $line-height-computed * 2;
   padding: ($line-height-computed * 2) ($grid-gutter-width * 2);
-  outline: none;
-}
 
-.play-button:hover {
-  background-color: rgba(black, .2);
-}
+  &:hover {
+    background-color: rgba(black, 0.2);
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.